### PR TITLE
Make the Ballerina project to adhere with VSCode tooling features

### PR DESCRIPTION
**Pull request by BuddhiWathsala**
## Purpose
Related to https://github.com/ballerina-platform/ballerina-standard-library/issues/1746

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina/Ballerina.toml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 name = "xslt"
-version = "@toml.version@"
+version = "2.0.0"
 distribution = "slbeta3"
 
 [[platform.java11.dependency]]
-path = "../native/build/libs/xslt-native-@project.version@.jar"
-version = "@project.version@"
+path = "../native/build/libs/xslt-native-2.0.0-SNAPSHOT.jar"
+version = "2.0.0-SNAPSHOT"
```
Comment (ThisaruGuruge) [inline ballerina/Ballerina.toml:9]: We don't need the version here, since we directly provide the jar path.

File: ballerina/Ballerina.toml [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 
 [[platform.java11.dependency]]
-path = "./lib/jakarta.activation-@jakarta.activation.version@.jar"
+path = "./lib/jakarta.activation-1.2.2.jar"
 groupId = "com.sun.activation"
 artifactId = "jakarta.activation"
```
Comment (ThisaruGuruge) [inline ballerina/Ballerina.toml:14]: These too are not needed.

File: ballerina/Ballerina.toml (hunk 1 above)
Comment (BuddhiWathsala) [reply ballerina/Ballerina.toml]: Fixed in https://github.com/ballerina-platform/module-ballerina-xslt/pull/265/commits/2ceaf4f54619639193e9bcfce4cbe37e1532dfbd

File: ballerina/Ballerina.toml (hunk 2 above)
Comment (BuddhiWathsala) [reply ballerina/Ballerina.toml]: Fixed in https://github.com/ballerina-platform/module-ballerina-xslt/pull/265/commits/2ceaf4f54619639193e9bcfce4cbe37e1532dfbd

File: ballerina/build.gradle [hunk 3]
```diff
@@ -73,16 +76,23 @@ task updateTomlFiles {
...
 }
 
-task revertTomlFiles {
+task commitTomlFiles {
     doLast {
-        ballerinaConfigFile.text = originalConfig
+        project.exec {
+            ignoreExitValue true
+            if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+                commandLine 'cmd', '/c', "git commit Ballerina.toml Dependencies.toml -m '[Automated] Update the native jar versions'"
```
Comment (ThisaruGuruge) [inline ballerina/build.gradle:91]: Single quotes doesn't work in Windows.
```suggestion
                commandLine 'cmd', '/c', "git commit Ballerina.toml Dependencies.toml -m \"[Automated] Update the native jar versions\""
```

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**